facts/Vagrantfile

```diff
@@ -91,12 +91,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     host.vm.provision "shell", path: "get_facts.sh"
     host.vm.provision "shell", inline: "/sbin/shutdown -h now"
   end
-  config.vm.define "centos-7-x86_64" do |host|
-    host.vm.box = "puppetlabs/centos-7.0-64-nocm"
-    host.vm.provision "file", source: "Gemfile", destination: "Gemfile"
-    host.vm.provision "shell", path: "get_facts.sh"
-    host.vm.provision "shell", inline: "/sbin/shutdown -h now"
-  end
+ config.vm.define "centos-7-x86_64" do |host|
```

Comment:
That's some left-over silliness from separating the CentOS 7 fixes from the Windows; will be fixed in a bit.